packager/mpd/base/adaptation_set.h

```diff
@@ -135,6 +135,9 @@ class AdaptationSet {
   /// @param id is the new ID to be set.
   void set_id(uint32_t id) { id_ = id; }
 
+  /// Get media info.
+  const MediaInfo& getMediaInfo() const { return media_info_;}
```

Comment:
s/getMediaInfo/media_info/ to be consistent.

Reply:
removed mediainfo from adaptation set